docs/rules/jsx-curly-spacing.md

```diff
@@ -135,6 +221,41 @@ When `"never"` is used and `objectLiterals` is `"always"`, the following pattern
 
 Please note that spacing of the object literal curly braces themselves is controlled by the built-in [`object-curly-spacing`](http://eslint.org/docs/rules/object-curly-spacing) rule.
 
+### Shorthand options
```

Comment:
I'd say we might want to deprecate them (string options are horrible for extensibility), but there's not really any pressing need to.